docs/src/main/asciidoc/security.adoc

```diff
@@ -55,35 +55,39 @@ Please see xref:security-built-in-authentication.adoc#mutual-tls[Mutual TLS Auth
 
 === OpenID Connect
 
-`quarkus-oidc` extension provides a reactive, interoperable, multi-tenant enabled OpenID Connect adapter which supports `Bearer Token` and `Authorization Code Flow` authentication mechanisms.
+OpenID Connect (OIDC) is an open authentication protocol that enables client applications to verify the identity of a user when that user tries to connect to a secure server, such as HTTPS.
+OIDC uses authorization and authentication mechanisms on OAuth 2.0 compliant authorization servers and builds identities that uniquely identify users.
 
-`Bearer Token` mechanism extracts the token from HTTP `Authorization` header. 
-`Authorization Code Flow` mechanism uses OpenID Connect Authorization Code flow. It redirects the user to IDP to authenticate and completes the authentication process after the user has been redirected back to Quarkus by exchanging the provided code grant for ID, access and refresh tokens.
+In Quarkus, the OIDC extension, `quarkus-oidc`, provides a reactive, interoperable, multitenant-enabled OIDC adapter that supports `Bearer Token` and `Authorization Code Flow` authentication mechanisms.
 
-ID and access `JWT` tokens are verified with the refreshable `JWK` key set but both JWT and opaque (binary) tokens can be introspected remotely. 
+The `Bearer Token` mechanism extracts the token from the HTTP `Authorization` header. 
+The `Authorization Code Flow` mechanism uses OIDC authorization code flow, which redirects you to an identity provider (IDP) to authenticate the identity of a user.
+Next, after you are redirected back to Quarkus, the mechanism completes the authentication process by exchanging the provided code grant for ID, access, and refresh tokens.
```

Comment:
This is a good introducstion to the code flow, nice work, but now adding a diagramm vsualizing it would be great :-).
Please note, in the diagramm, instead of listing the added like `/authorization` please say `Authorization Endpoint`, and instead of `/token` - `Token Endpoint`

Reply:
I've also simplified a bit `The `Authorization Code Flow` mechanism uses OIDC authorization code flow, which redirects` as `The `Authorization Code Flow` mechanism redirects`